customer-terminate-service/app/src/test/java/se/inera/intyg/cts/application/service/ExportServiceTest.java

```diff
@@ -0,0 +1,67 @@
+package se.inera.intyg.cts.application.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static se.inera.intyg.cts.testutil.CertificateTestDataBuilder.certificates;
+import static se.inera.intyg.cts.testutil.TerminationTestDataBuilder.defaultTerminationEntity;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import se.inera.intyg.cts.domain.service.CollectExportContent;
+import se.inera.intyg.cts.domain.service.ExportPackage;
+import se.inera.intyg.cts.infrastructure.integration.GetCertificateBatchFromMemory;
+import se.inera.intyg.cts.infrastructure.integration.GetCertificateTextsFromMemory;
+import se.inera.intyg.cts.infrastructure.integration.IntegrationCertificateBatchRepository;
+import se.inera.intyg.cts.infrastructure.persistence.JpaCertificateRepository;
+import se.inera.intyg.cts.infrastructure.persistence.JpaCertificateTextRepository;
+import se.inera.intyg.cts.infrastructure.persistence.JpaTerminationRepository;
+import se.inera.intyg.cts.infrastructure.persistence.repository.InMemoryCertificateEntityRepository;
+import se.inera.intyg.cts.infrastructure.persistence.repository.InMemoryCertificateTextsEntityRepository;
+import se.inera.intyg.cts.infrastructure.persistence.repository.InMemoryTerminationEntityRepository;
+import se.inera.intyg.cts.infrastructure.service.CreateEncryptedZipPackage;
+
+class ExportServiceTest {
+
```

Comment:
They are implemented once and can be used thoughout testing. You only wire up the data you want to work with. You don't need, for each test to consider what to mock and what behaviours should be used for each test. See other comments for more reasoning on using this instead of mocking.